src/NCrypt.Tests/NCrypt.cs

```diff
@@ -15,4 +15,26 @@ public void OpenStorageProvider()
         {
         }
     }
+
+    [Fact(Skip = "fails, and I don't know how to fix it")]
```

Comment:
It always fails. I've asked NCrypt folks for help but haven't gotten good help yet. I'm guessing I just don't know how to code up the test right.